### PR TITLE
New package: polychromatic & dep (python3-colour)

### PR DESCRIPTION
**Pull request by dmarto**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/python3-colour/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-colour'
+pkgname=python3-colour
+version=0.1.5
+revision=1
+wrksrc="${pkgname#*-}-${version}"
```
[the remark this answers is not in the document]
Comment (classabbyamp) [reply srcpkgs/python3-colour/template:5]: `${pkgname#python3-}` is a common pattern in python packages for $wrksrc

[the remark this answers is not in the document]
Comment (dmarto) [reply srcpkgs/python3-colour/template]: updated

**Comment by dmarto** [comment]
bump`d

File: srcpkgs/python3-colour/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmarto) [reply srcpkgs/python3-colour/template]: thanks to the recant updates to xbps-src, the line is fully removed now

File: srcpkgs/polychromatic/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+version=0.8.0
+revision=1
+build_style=meson
+hostmakedepends="ninja sassc which gettext"
+depends="python3-colorama python3-setproctitle python3-requests
+ python3-colour python3-PyQt5-svg python3-PyQt5-webengine"
+short_desc="RGB lighting management software for GNU/Linux"
+maintainer="Martin Dimov <martin@dmarto.com>"
+license="GPL-3.0-or-later"
+homepage="https://polychromatic.app"
```
Comment (classabbyamp) [inline srcpkgs/polychromatic/template:12]:
```suggestion
homepage="https://polychromatic.app"
changelog="https://github.com/polychromatic/polychromatic/raw/master/CHANGELOG"
```

File: srcpkgs/polychromatic/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'polychromatic'
+pkgname=polychromatic
+version=0.8.0
+revision=1
+build_style=meson
+hostmakedepends="ninja sassc which gettext"
+depends="python3-colorama python3-setproctitle python3-requests
+ python3-colour python3-PyQt5-svg python3-PyQt5-webengine"
+short_desc="RGB lighting management software for GNU/Linux"
```
Comment (classabbyamp) [inline srcpkgs/polychromatic/template:9]:
```suggestion
short_desc="RGB lighting management software"
```
feel like that part is kinda redundant

Comment (dmarto) [reply srcpkgs/polychromatic/template]: Yea, I just copy pasted their tagline.

**Pull request merged**